chrome-extension/content.js

```diff
@@ -108,3 +115,23 @@ function customCreateElement(type, classList, innerHTML = '') {
   newElement.innerHTML = innerHTML;
   return newElement;
 }
+
+function addItem() {
+  newItem = {
+    sourceLink: 'www.googe.com', //TODO: get actual sourceLink
+    placesId: 'something', //TODO: get actual placesId, to-be implemented
+    userId: '5f050952f516f3570ee26724', //TODO: get actual userID
+    highlight: document.getElementById('highlight').value,
+    comment: document.getElementById('comment').value,
+  }
+  fetch('http://localhost:8080/item/addItem', {
```

Comment:
Optional for future: consider setting up a transpiler to convert the promise then/catch es6 to be compatible for more versions of Chrome. Optional since only 0.11% of Chrome versions in use don't support promises. Along this line of logic, only 0.66% of Chrome versions don't support `await` so you could also change it to that and plan to add a transpiler in the future? Or if broad browser support isn't your MVP goal then you could just change it now! Up to you.

[Example](https://www.coreycleary.me/setting-up-chrome-extensions-for-use-with-es6/)

Reply:
okay, I will change this to await and then open an issue for transpiler, to be done later.